server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueueFactory.java

```diff
@@ -81,10 +81,6 @@ private RabbitMQMailQueue attemptQueueCreation(MailQueueName mailQueueName) {
 
     private RabbitMQMailQueue getOrElseCreateLocally(MailQueueName name) {
         return Optional.ofNullable(mailQueueStoreLocal.get(name))
-            .orElseGet(() -> createLocally(name));
-    }
-
-    private RabbitMQMailQueue createLocally(MailQueueName name) {
-        return mailQueueStoreLocal.computeIfAbsent(name, mailQueueFactory::create);
+            .orElseGet(() -> mailQueueStoreLocal.computeIfAbsent(name, mailQueueFactory::create));
```

Comment:
How about only `return mailQueueStoreLocal.computeIfAbsent(name, mailQueueFactory::create);`

Reply:
sorry, You are right!